### PR TITLE
👫 Handle invitations display

### LINKED ISSUE #1443: Display invitees and invitor for member
On the [membership modal](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=2596%3A115078), we should display information about invitees and invitor.

### PR DESCRIPTION
**Pull request by jodator**
Closes #1443

### CODE REVIEW

File: packages/ui/dev/scripts/generators/generateMembers.ts [hunk 1]
```diff
@@ -51,8 +51,34 @@ const generateMember = (override?: KnownMember) => ({
   },
 })
 
-export type MemberMock = ReturnType<typeof generateMember>
+const inviteMember = (invitedById: string) => ({
```
Comment (p-sad) [inline packages/ui/dev/scripts/generators/generateMembers.ts:54]: Looks almost identical to generateMember; do we care enough about mocks generators to extract common code into a function?

**Review by p-sad — APPROVED** [review]
Looks good

File: packages/ui/dev/scripts/generators/generateMembers.ts (hunk 1 above)
Comment (jodator) [reply packages/ui/dev/scripts/generators/generateMembers.ts:54]: I'll fix that.

Comment (jodator) [reply packages/ui/dev/scripts/generators/generateMembers.ts:54]: And entry is a bit different

Comment (jodator) [reply packages/ui/dev/scripts/generators/generateMembers.ts:54]: Minor cleanup in: [`be0e88b`](https://github.com/Joystream/pioneer/pull/1466/commits/be0e88bd9e4b7db235e707c16e91a55d61b29eec).

**Pull request merged**